src/auth/__mocks__/auth.controller.ts

```diff
@@ -0,0 +1,17 @@
+import { faker } from '@faker-js/faker';
+
+export class mockAuthController {
```

Comment:
class should be in `PascalCase` so:
```suggestion
export class MockAuthController {
```